### PR TITLE
Require chormatic ci

### PR DESCRIPTION
**Pull request by akomiqaia**
## Describe your changes
add visual regression testing

## Checklist before requesting a review

- [x] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review.
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -34,4 +34,5 @@ jobs:
           storybookBaseDir: apps/o3-storybook
           exitOnceUploaded: true
           onlyChanged: true
-          exitZeroOnChanges: true
+          autoAcceptChanges: "main" # 👈 Option to accept all changes on maine
```
Comment (frshwtr) [inline .github/workflows/chromatic.yml:37]:
```suggestion
          autoAcceptChanges: "main" # 👈 Option to accept all changes on main
```

File: .github/workflows/chromatic.yml [hunk 2]
```diff
@@ -34,4 +34,5 @@ jobs:
...
+          # exitZeroOnChanges: true
```
Comment (frshwtr) [inline .github/workflows/chromatic.yml:38]: If this is no longer needed can we remove it?

File: components/o3-button/stories/core/button.stories.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ export default {
 	decorators: [
 		Story => (
 			<div data-o3-brand="core">
+				test
```
Comment (frshwtr) [inline components/o3-button/stories/core/button.stories.tsx:15]:
```suggestion
```

**Review by notlee — APPROVED** [review]

**Comment by notlee** [comment]
Approved 😄 Happy to take another look when you've added some documentation on running Chromatic and getting approvals – either here or in another PR

**Pull request merged**